### PR TITLE
protein-translation: add to track

### PR DESCRIPTION
**Pull request by Smarticles101**

### CODE REVIEW

**Comment by Smarticles101** [comment]
This should be ready for review now. Doesn't look like the javascript track is super active, so review/merge when you can :) I'll try to contribute some more, especially since I need to brush up on my javascript.

File: exercises/protein-translation/example.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+function translate(rnaStrand) {
+  var proteins = [];
+
+  if (rnaStrand) {
+    for (var i = 0; i < rnaStrand.length; i+=3) {
+      var protein = getProtein(rnaStrand.substring(i, i + 3));
+
+      if (protein) {
+
+        if (protein === "STOP")
```
Comment (matthewmorgan) [inline exercises/protein-translation/example.js]: We're using the AirBnb JS style guide on this track, so your `if` statements should have [braces please](https://github.com/airbnb/javascript#blocks--braces)!

Comment (Smarticles101) [reply exercises/protein-translation/example.js]: Fixed!

Comment (matthewmorgan) [reply exercises/protein-translation/example.js]: Thanks @Smarticles101 !

**Pull request merged**